### DISCUSSION TITLE
Nuxt module Authors HMR not working as expected

### DISCUSSION THREAD

**Discussion by cpt-n3mo** [nuxt/nuxt#28185] [Questions]
Good day..

I was wondering if some more ppl have trouble when creating a nuxt module, 
i added  some component dirs and pages, but  hmr is not working properly in these pages and components, it seems to do a hmr maybe 2x and then i need to restart the nuxt app, page reloading does not work either. 
i checked in dev tools of firefox and chrome and caching is disabled. but it seems that nuxt itself might be caching.

i dont wat to create a bug report just yet, thinking i might not have this configured correctly.

kind regards

**Comment by dulnan** [comment]
I also ran into this issue. What solved it for me is adding the following in the playground's nuxt.config.ts:

```typescript
export default defineNuxtConfig({
  vite: {
    server: {
      watch: {
        usePolling: true,
      },
    },
  },
})
```
